backend/tests/test_seed_qa.py

```
@@ -0,0 +1,253 @@
+        mock_settings.app_env = "production"
+        with pytest.raises(SeedGuardError):
+            assert_seed_allowed()
+
+
+@pytest.mark.unit
+@pytest.mark.parametrize("env", ["qa", "development", "test"])
+def test_assert_seed_allowed_permits_non_production(env):
+    """Non-prod environments proceed without raising."""
+    with patch("app.services.seed_qa.settings") as mock_settings:
+        mock_settings.app_env = env
+        assert_seed_allowed() is None
```

Ruff (0.15.15)

[warning] Pointless comparison. Did you mean to assign a value? Otherwise, prepend `assert` or remove it.

(B015)